docs/jwst/faq.rst

```diff
@@ -0,0 +1,215 @@
+.. _faq:
+
+Frequently Asked Questions
+==========================
+
+How do I install the JWST pipeline?
+-----------------------------------
+
+The latest public release can be installed via pip.
+::
+
+	$ pip install jwst
+
+To work with the absolute most up-to-date version of the pipeline, the ``main``
+branch of the Github repository can be installed.
+
+::
+
+	$ pip install git+https://github.com/spacetelescope/jwst
+
+To work with a different branch in the Github repository, install
+that branch using its name, e.g. ``patch1``.
+
+::
+
+	$ pip install git+https://github.com/spacetelescope/jwst@patch1
+
+See the `Github README <https://github.com/spacetelescope/jwst>`_ for more
+detailed installation instructions.
+
+How do I run the JWST pipeline?
+-------------------------------
+
+There are two options for running the JWST pipeline or individual pipeline steps.
+
+In Python
+^^^^^^^^^
+Pipelines and steps can be imported, configured, and run in a Python session.
+See `here <https://jwst-pipeline.readthedocs.io/en/latest/jwst/introduction.html#running-from-within-python>`__ for more information.
+
+On the command line
+^^^^^^^^^^^^^^^^^^^
+
+Use the `strun<strun_command_line>` command.
+
+Both of these options access the same underlying code, the choice of which to
+use is a matter of personal preference and convenience.
+
+.. _stpipe_list:
+
+How do I find out which pipelines and steps exist and are available to run?
+---------------------------------------------------------------------------
+
+Running the pipeline requires you to know the exact class name of the pipeline
+or step that you wish to run (e.g Detector1Pipeline). Some pipelines have shorter
+aliases that can be used interchangeably with their full class names. The command
+line tool
+::
+
+   $ stpipe list
+
+will show all available pipelines and steps and their aliases. Additionally, a
+list of all pipelines and aliases is available `here <https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/main.html#pipelines>`__.
+
+What files does the pipeline run on?
+------------------------------------
+
+The pipeline either takes a single ``.fits`` file, or an association (``asn.json``)
+file which represents a group of related exposures, as input to process.
+
+
```

Comment:
Could add a link here to this page: https://jwst-pipeline.readthedocs.io/en/latest/jwst/data_products/product_types.html